lib/puppet/application/master.rb

```diff
@@ -172,7 +172,7 @@ def compile
         raise "Could not compile catalog for #{options[:node]}"
       end
 
-      puts PSON::pretty_generate(catalog.to_resource, :allow_nan => true, :max_nesting => false)
+      puts JSON::pretty_generate(catalog.to_resource, :allow_nan => true, :max_nesting => false)
```

Comment:
I don't think this will work, JSON::pretty_generate only accepts arrays or hashes